Area51/Sources/SubredditsViewController.swift

```diff
@@ -166,8 +176,24 @@ extension SubredditsViewController: UITableViewDataSource, UITableViewDelegate {
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         dataSource.loadMoreIfNeeded(currentIndex: indexPath.row)
         let cell = tableView.dequeueReusableCell(withIdentifier: "SubredditCell")!
+
+        //set placeholder image
+        if let image = UIImage(named: "AppIcon") {
+            cell.imageView?.image = image
+        }
+
         if let subreddit = getSubreddit(forIndexPath: indexPath) {
             cell.textLabel?.text = subreddit.displayName
+
+            if let imageURL = subreddit.iconImgURL {
+
+                ImageFetcher.image(forURL: imageURL) { [weak imageView = cell.imageView, weak self] (image, url) in
```

Comment:
Can we just have the tableviewCell imageView use this instead? https://github.com/kgellci/Area51/blob/master/ImageService/Sources/NetworkImageView.swift

It already handles loading and setting the image. 

Also, do we need to resize the image? We can just set the ImageView to use aspect fit or aspect fill as the content mode.

Reply:
When using the image without resizing, I ran into the following issue.
The image would resize when touched. I could not make it work using aspectFit or aspectFill.
<img width="342" alt="Screen Shot 2019-06-03 at 10 33 40 PM" src="https://user-images.githubusercontent.com/12742157/58820295-d8d6b980-864f-11e9-891e-63d8de60c562.png">

Regarding the NetworkImageView, should I be able to use that custom imageView without creating a custom table view cell? Is it better to use a custom tableview cell as it might also help with the resizing issue?

Reply:
Yah, if you make a custom cell, it should simplify all the logic. You can use the NetworkImageView and give it a width + height constraint.

Reply:
Added a custom imageView and a label to the prototype cell.
Used NetworkImageView to load logo images.